users/admin.py

```diff
@@ -0,0 +1,51 @@
+"""
+Settings for admin site
+"""
+from django.contrib import admin
+from django.contrib.auth.admin import UserAdmin as DefaultAdmin
+
+# from .forms import CustomUserChangeForm, CustomUserCreationForm
```

Comment:
This needs to be removed. Also, All the commented lines should be removed

Reply:
Done